Appraisals

```diff
@@ -1,10 +1,13 @@
 # frozen_string_literal: true
 
 appraise "ar-60" do
-  # we are using this version as default in gemspec
-  # gem "activerecord", "~> 6.0.0"
+  gem "activerecord", "~> 6.0.0"
 end
 
 appraise "ar-61" do
   gem "activerecord", "~> 6.1.1"
 end
+
+appraise "ar-70" do
+  # latest version
```

Comment:
```suggestion
  gem "activerecord", "~> 7.1.2"
```

Don't we need it? Alternatively, I think we can bump `activerecord` version in gemspec

https://github.com/discourse/prometheus_exporter/blob/5c994acd1eb9f96f3dcc71c8cdf8cf41490b11e8/prometheus_exporter.gemspec#L39

Reply:
no idea I was just fiddling with buttons till I got it working :) I guess we need to add 71 and 70 to the matrix